src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -216,10 +219,10 @@ public String updateDeviceMember(ManagerDeviceMemberElement deviceElement, Strin
         deviceMember.setId(Long.parseLong(deviceElement.getId()));
         deviceMember.setName(name);
```

Comment:
You can simply set the default values here in the deviceMember and let it flow down. This way you dont need to add a flag anywhere. Also this will clearly be isolated and used on the SDK implementation path as opposed to the rest path.

Reply:
Also do the same thing for the create path in the SDK.

Reply:
This won't work when we do the sync of DA, it will update with the default values, rather then take what the OSC gives and update the rest from DB. will see how not to have impact on REST.

Reply:
I plan to update as follows ==> read from DB, if the version info is NULL, then set with default. otherwise pick up values from DB. I will add this in SDK implementation as proposed with DB read.

Reply:
The implementation is inline with your comments as above.

Reply:
@emanoelxavier , This version is tested on deployed Build#85 build and all tests are passing. The default values are populated as expected.